lib/regular_expression/cfg.rb

```diff
@@ -177,12 +178,12 @@ def to_dot(graph)
           block.exits.each do |block_exit|
             successor = nodes[label_map[block_exit.label]]
             attributes = {}
-            if (kind = block_exit.metadata[:kind])
-              attributes["color"] = { true_edge: "green", false_edge: "red" }[kind]
-            end
-            if (probability = block_exit.metadata[:probability])
-              attributes["penwidth"] = 1 + probability * 4
-            end
+            # if (kind = block_exit.metadata[:kind])
+            #   attributes["color"] = { true_edge: "green", false_edge: "red" }[kind]
+            # end
+            # if (probability = block_exit.metadata[:probability])
+            #   attributes["penwidth"] = 1 + probability * 4
+            # end
```

Comment:
What was wrong here?

Reply:
It was throwing `hash key "color" is not a Symbol`

Reply:
Maybe it's because I have an old ruby version

Reply:
It shouldn't be anything but 3.0.

https://github.com/kddnewton/regular_expression/blob/main/.ruby-version